user-docs/latest/content-manager/writing-content.md

```diff
@@ -0,0 +1,63 @@
+| Media       | 1. Click the media area. <br> 2. Choose an asset from the Media Library, or click the **Add more assets** button to add a new file to the Media Library. <br><br> 💡 It is possible to drag and drop the chosen file in the media area.                                                                                                                                        |
+| JSON        | Write your content, in JSON format, in the code textbox.                                                                                                                                                                                                                                                                                                                     |
+| UID         | Write a unique identifier in the textbox. A "Regenerate" button, displayed on the right of the box, allows to automatically generate a UID based on the content type name.                                                                                                                                                                                                   |
+
+### Components
+
+Components are a combination of several fields. Writing their content works exactly like for independant fields. They differ in their display, since the fields of a component are grouped together.
+
+![Writing content for a component](../assets/content-manager/edit-view_component.png)
+
+### Dynamic zones
+
+Dynamic zones are a combination of components, which themselves are composed of several fields. Writing the content of a dynamic zone requires additional steps in order to access the fields.
+
+1. Click on the ![icon add to content](../assets/content-manager/icon_add3.png) **Add to content** button.
+2. Choose a component available for the dynamic zone.
+3. Fill in the fields of the component.
+
+![Writing content for a dynamic zone](../assets/content-manager/edit-view_dynamic-zone1.png)
+![Writing content for a dynamic zone](../assets/content-manager/edit-view_dynamic-zone2.png)
+
+In contrary to regular fields and components, dynamic zones can also be reordered or deleted directly in the edit view, using buttons displayed in the top right corner of the dynamic zone area.
+
+- Use the arrow button ![icon reorder](../assets/content-manager/icon_reorder.png) to reorder components in your dynamic zone.
+- Use the trash button ![icon delete](../assets/content-manager/icon_delete3.png) to delete a component from your dynamic zone.
+
+::: tip NOTE
+Unlike regular fields, the order of the fields and components inside a dynamic field is important. It should correspond exactly to how end-users will read/see the content.
+:::
+
+## Collaborating on content writing
+
+Contents created with Strapi may be edited by several admin panel users. Since these contents cannot be versioned, and to prevent any content loss, Strapi automatically informs users of concurrent edition situations.
```

Comment:
"Strapi automatically informs users of concurrent edition situations" to confirm but I think this will not be developed by the release of the Relational Fields, when this doc will be released.
To confirm with @Aurelsicoko .

Reply:
I'm fairly certain it made it in (at least I've seen the PRs for it, trying to follow the product teams merges is... difficult sometimes)

Reply:
@sam-pires will give you more details :)

Reply:
good catch 

High probability that Concurrent editing won't be released in 3.4.
Indeed a big chunk or it has been develop. But not all devs agree on the way the remaining part should be done, plus some says it won't be compatible with i18n, others say it will. We need to discuss it but we wait for RF to. be finished.
We'll discuss it on Monday or Wednesday